### PR TITLE
[CP Stg] Wrap AddPersonalBankAccount with KeyboardAvoidingView

### PR DESCRIPTION
**Pull request by nickmurray47**
### Details
Keyboard being hidden when attempting to add a personal bank account from 

### Fixed Issues

$ [GH_LINK](https://github.com/Expensify/App/issues/6666)

### Tests / QA Steps
1. Launch the app and login
1. Navigate to Settings > Payments > Add payment method and select "Bank account"
1. Verify the Plaid modal is presented and login with Chase test credentials.
1. Select an account from the list
1. Enter incorrect or correct password
2. Verify the "Save and continue" button displayed above the keyboard or scrollable to the keyboard

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [ ] Android

#### Web

Screen Shot 2021-12-09 at 10 57 23 AM

### CODE REVIEW

File: src/pages/AddPersonalBankAccountPage.js [hunk 1]
```diff
@@ -24,21 +25,23 @@ const defaultProps = {
...
-            plaidLinkOAuthToken={props.plaidLinkToken}
-            isPasswordRequired
-        />
+        <KeyboardAvoidingView>
+            <HeaderWithCloseButton
+                title={props.translate('bankAccount.addBankAccount')}
+                onCloseButtonPress={Navigation.dismissModal}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.goBack()}
+            />
```
Comment (parasharrajat) [inline src/pages/AddPersonalBankAccountPage.js:34]: This should be outside the `KeyboardAvoidingView`.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/pages/AddPersonalBankAccountPage.js (hunk 1 above)
Comment (nickmurray47) [reply src/pages/AddPersonalBankAccountPage.js:34]: Why would that be?

Comment (parasharrajat) [reply src/pages/AddPersonalBankAccountPage.js:34]: Headers are fixed all the time. we don't want to adjust them based on the keyboard. It makes the code more clear to just wrap the main content and the whole app follows that pattern. :cat:

Comment (nickmurray47) [reply src/pages/AddPersonalBankAccountPage.js:34]: The headers are fixed, but every component I've looked at in the app follows the pattern of having `KeyboardAvoidingView` outside of the `Header`

Comment (parasharrajat) [reply src/pages/AddPersonalBankAccountPage.js:34]: Oh, man. I am just confused. You are correct. I was thinking we put the header outside but that was Scrollview. Sorry for the confusion.

Comment (nickmurray47) [reply src/pages/AddPersonalBankAccountPage.js:34]: All good! Appreciate the review

**Comment by parasharrajat** [comment]
Unfortunately, I can't run VBA flow. Don't have access to _Chase test credentials_

**Comment by nickmurray47** [comment]
Thanks Marc!

**Comment by marcaaron** [comment]
@parasharrajat for future reference the Chase credentials should work when testing against the staging "secure API" (where the Plaid sandbox is running and all the bank account logic goes through a separate endpoint, but this would need to be set up in the `.env` file). 

The credentials are the same ones recommended by Plaid and described in this [doc](https://plaid.com/docs/sandbox/test-credentials/).

Login: `user_good`
Password: `pass_good`

**Comment by parasharrajat** [comment]
Thanks, @marcaaron, I will keep this as a reference.

**Comment by marcaaron** [comment]
Passes QA steps for Personal Bank Account flow. 

I am a little curious if we can try to make this a feature of the `ReimbursementAccountForm` instead of having to add it to `AddPlaidBankAccount` which is used by both VBA and PBA flow. VBA already has a `KeyboardAvoidingView` here:

https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/ReimbursementAccountPage.js#L209

But the presence of two of them doesn't seem to cause any issues for the VBA flow so it's NAB for me.

**Review by marcaaron — APPROVED** [review]

**Comment by nickmurray47** [comment]
> I am a little curious if we can try to make this a feature of the ReimbursementAccountForm instead of having to add it to AddPlaidBankAccount which is used by both VBA and PBA flow. VBA already has a KeyboardAvoidingView here:

Good point, happy to go for the refactor as soon as I get my environment working

**Pull request merged**